examples/specs/layer_bar_annotations.vl.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "The PM2.5 value of Beijing observed 15 days, highlighting the days when PM2.5 level is hazardous to human health",
+  "data": {
+    "values": [
+      {"Day": 1, "Value": 50},
+      {"Day": 2, "Value": 110},
+      {"Day": 3, "Value": 70},
+      {"Day": 4, "Value": 50},
+      {"Day": 5, "Value": 80},
+      {"Day": 6, "Value": 150},
+      {"Day": 7, "Value": 140},
+      {"Day": 8, "Value": 130},
+      {"Day": 9, "Value": 390},
+      {"Day": 10, "Value": 200},
+      {"Day": 11, "Value": 70},
+      {"Day": 12, "Value": 50},
+      {"Day": 13, "Value": 100},
+      {"Day": 14, "Value": 150},
+      {"Day": 15, "Value": 130}
+    ]
+  },
+  "layer": [
+    {
+      "mark": "bar",
+      "encoding": {
+        "x": {"field": "Day", "type": "ordinal"},
+        "y": {"field": "Value", "type": "quantitative"}
+      }
+    },
+    {
+      "mark": "bar",
+      "transform": [{
+        "calculate": "datum.Value >= 300 ? 300 : datum.Value",
```

Comment:
Technically this group of marks still generates N bars under the hood, which is semantically wrong.  We only want to draw one bar here. 

Please use filter and calculate baseline = 300 without condition.